### ISSUE TITLE
Everyone needs to help!

### ISSUE THREAD

**Issue by firebug11** [Maxx53/SCMBot#421]
There are lots of questions and issues of people not knowing the optimum settings for Maxx's great bot and some that do not have the time to figure them out!

The community that uses this bot please can you help out and add to this post if you have found the optimum settings for what you scan (You do not have to give the specific item away because of competition) but the more you help the better the chance of you finding a way to beat the other people scanning your item

Post what works for you in the different categories:

The item:On the market(Your average item) / Not on the market (High tier knives)
Mode: Html/JSON
Delay:??
Hosts:Yes/No

Thank you,

**Comment by firebug11** [comment]
I will kick things off by saying what i had success with:

The item:On the market
Mode: Html
Delay:350ms
Hosts:Yes but all of them with under 40ms

The item: Not on the market 
Mode: JSON
Delay:100
Hosts:Yes but all of them with under 40ms

**Comment by mtzi** [comment]
The item:On the market
Mode: Html
Delay:400ms (2 items) OR 1500-2000 (5 items)
Hosts:Yes but all of them with under 60ms (15 hosts 45-55 ms)

however i had little success due to the fact i looked high tier market knives (250$) for prices under 20% of their normal value (happens once a month if lucky on each knife)

**Comment by Alex1os** [comment]
this was the latest rare knife I've gotten 

The item: Not on the market
Mode: Html
Delay: 2500 ms
Hosts: Yes but all of them with under 20ms (15 hosts)
Request Delay: 100ms

this worked for 4 high tier knives with settings 1000ms-3000ms

Lg Alex

**Comment by sebmins** [comment]
My delay doesn't change anything, if its on 3000 or 1 it still measures it every 20 seconds am I doing something wrong?

**Comment by mtzi** [comment]
in settings, play with delay between requests maybe?

**Comment by sebmins** [comment]
How often in seconds does the bot scan for you?

**Comment by mtzi** [comment]
i have delay between requests 0, item delay 400, getting 3 pings every 4 sec. 
like 11:55:54 - 11:55:55 - 11:55:57 looks like this, 1-1-2 sec delay after each ping to the next, i have only 2 items tho, if u have many items the delays bug each other, if u search for 15 items its normal that 5 are searched normal and 10 every 1 min or so

**Comment by sebmins** [comment]
Yeah if I only use 1 item I can get it to scan every second or so but I am on like 15 and it scans every 20 seconds can I decrease this? I used to use the steam trader helper and I would have 90+ items scanning and I would buy an item every day. with the scmbot im not buying any items in 5+ days, does anyone know how often the steam trader helper would scan? 
Ps I scanned on slow

**Comment by mtzi** [comment]
no idea, i dont think i can help you with this. sorry.
usually with scmbot u scam max 9 items, every 3-4 sec.
what happens is bot look at 1 item, then waits the "delay between requests" to send next request, so if it scans 10 items, every 300ms it passes the "item delay" which is low for u, so the rest in the list are not scanned, because the list goes back to start
example of 15 items, 1->2->3->4->5->6->7 (lets say here the item delay is  filled, its been 3 sec since item 1, so it goes back to 1, or any random in the list)->2->5->8->1->12->15
thats why u get this low scan every 20 sec

**Comment by sebmins** [comment]
Ok thanks mate :smiley_cat:
